src/mantine-core/src/RingProgress/RingProgress.tsx

```diff
@@ -23,7 +24,7 @@ export interface RingProgressProps
   roundCaps?: boolean;
 
   /** Ring sections */
-  sections: { value: number; color: MantineColor; tooltip?: React.ReactNode }[];
+  sections: CurveData[];
```

Comment:
This change will remove type information from props table and users will not know what properties `sections` supports.

Reply:
sorry. I didn't consider what you said. I will close the issue.